src/pybind/mgr/balancer/module.py

```diff
@@ -294,10 +294,10 @@ class Module(MgrModule):
             'runtime': True,
         },
         {
-            'name': 'upmap_max_iterations',
```

Comment:
yeah, that might be a problem if you have any customized upmap_max_iterations values.
I think we should definitely update the PendingReleaseNotes accordingly to make sure user is aware of this change, but let's get some feedback from @liewegas  first.

Reply:
In case this ends up being important, we might want to prevent forcing ceph-ansible, Rook and DeepSea to implement some logic to do the upgrade automatically

Reply:
@sebastian-philipp I don't quite follow you. The only thing which I think matters is that
you should delete the old upmap_max_iterations key and then copy the value to the new upmap_max_optimizations instead in case you have any customized upmap_max_iterations value.

Reply:
I think a release note is sufficient.. very few users will have changed this.

eventually we may want to have a way for the mon (or individual mgr modules) to do these conversion during the upgrade automatically...

Reply:
@liewegas  PendingReleaseNotes refreshed.